scripts/queue_manager.R

```diff
@@ -37,15 +36,21 @@ source(here::here("funs","queue_fun.R"))
 source(here::here("funs","download_fun.R"))
 source(here::here("scripts","test_functions.R"))
 
+# setup logging
+pid <- Sys.getpid()
+logfile <- file.path(get_database_dirname(),paste0("queue.log"))
+log_threshold(DEBUG)
```

Comment:
@jspijker , dp you want to store all the debug logging?

Reply:
It's far less than the trace messages